### ISSUE TITLE
Prefill not displaying properly

### ISSUE THREAD

**Issue by SilviaAmAm** [open-formulieren/open-forms#577]
Issue described in Taiga issue 10.

![Weergave leeg prefill veld](https://user-images.githubusercontent.com/19154114/130224623-bece868a-3684-4f15-94ad-69f1f53d3bd1.JPG)

**Comment by SilviaAmAm** [comment]
I tried reproducing this issue by mocking the retrieval of the data from Haal centraal. I used this data: https://github.com/open-formulieren/open-forms/blob/master/src/openforms/prefill/contrib/haalcentraal/tests/files/ingeschrevenpersonen.999990676-full.json (changing `huisletter` to "A" and `huisnummertoevoeging` to "bis"). However, this seems to display correctly:
![Screenshot from 2021-09-02 15-47-09](https://user-images.githubusercontent.com/19154114/131855592-d251bb25-1dde-4b79-9adc-bfad16076f02.png)

To investigate further it would be good to see which prefill is used to retrieve the data and see if the data comes back in a different format from the one expected.
